crates/biome_js_analyze/src/lint/complexity/no_useless_catch.rs

```diff
@@ -113,4 +133,86 @@ impl Rule for NoUselessCatch {
             )),
         )
     }
+
+    fn action(ctx: &RuleContext<Self>, _: &Self::State) -> Option<JsRuleAction> {
+        let node = ctx.query();
+
+        let body = node.body()?;
+        let stmts = body.statements();
+        let expressions: Vec<JsExpressionStatement> = stmts
+            .iter()
+            .map(|stmt| stmt.as_js_expression_statement().unwrap().clone())
```

Comment:
Don't use `unwrap`, it's unsafe and panics. Use another method. In this case you could do this:

```rs
.and_then(|stmt| stmt.as_js_expression_statement())
.cloned()
```